_dashboards/index.md

```diff
@@ -17,50 +17,43 @@ OpenSearch Dashboards is the preferred visualization platform for analyzing data
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards-example.png" alt="Screenshot of Dashboards interface" />
 
 # OpenSearch Dashboards for your use case
 
 OpenSearch Dashboards makes it easy to visualize data for a number of use cases such as log analytics, application search, enterprise search, and more. Here are a few examples.  
 
 <table>
-<thead>
-</thead>
 <tbody>
 <tr>
-<td>Visualize log and trace data with interactive log analytics<img src="{{site.url}}{{site.baseurl}}/images/trace_log_correlation-web.jpg" alt="Log analytics" width="400" style="float: left; margin-right: 15px;"/></td>
-<td>Centralize and analyze logs for real-time threat detection and incident management<img src="{{site.url}}{{site.baseurl}}//images/detection.png" alt="Anomaly detection" width="400" style="float: left; margin-right: 15px;"/></td>
+<td style="text-align: center; vertical-align: top;">Visualize log and trace data with interactive log analytics<img src="{{site.url}}{{site.baseurl}}/images/trace_log_correlation-web.jpg" alt="Log analytics"/></td>
+<td style="text-align: center; vertical-align: top;">Detect and mitigate issues faster with anomaly detection<img src="{{site.url}}{{site.baseurl}}/images/anomaly-detection.png" alt="Anomaly detection"/></td>
 </tr>
 <tr>
-<td>Quickly find relvant data within your applications, websites, and data catalogs<img src="{{site.url}}{{site.baseurl}}/images/drag-and-drop.png" alt="Data overview" width="400" style="float: left; margin-right: 15px;"/></td>
-<td>Find and fix problems and improve application health at a glance<img src="{{site.url}}{{site.baseurl}}/images/dashboards5.PNG" alt="Event analysis" width="400" style="float: left; margin-right: 15px;"/></td>
+<td style="text-align: center; vertical-align: top;">Use observability to monitor and issue alerts<img src="{{site.url}}{{site.baseurl}}/images/observability.png" alt="Data overview" /></td>
+<td style="text-align: center; vertical-align: top;">Unlock real-time search, monitoring, and analysis of business and operational data<img src="{{site.url}}{{site.baseurl}}/images/real-time-insights.png" alt="Real-time search, monitoring, and analysis" /></td>
 </tr>
 </tbody>
 </table>
 
-# Install and create OpenSearch Dashboards
+# Install and try OpenSearch Dashboards
 
 After installing OpenSearch, you can install OpenSearch Dashboards, with different installation methods to choose. For more information, see [OpenSearch Installation Instructions](https://opensearch.org/docs/latest/opensearch/install/index/).
 
-You can follow these steps to create a dashboard using sample data. 
+You can try OpenSearch Dashboards' features in the [OpenSearch Dashboards Playground](https://playground.opensearch.org/app/home). Choose **OpenSearch Dashboards > Dashboards > Create dashboard** and explore the sample Global Flight, Web Traffic, and Revenue dashboards. You can follow these steps to create a dashboard using sample data. 
```

Comment:
This should be two separate sections (like before):
1. Try dashboards with playground
 Choose **OpenSearch Dashboards > Dashboard**. Choose a dashboard from the existing ones and explore.
3. Try dashboards after installing it locally
(the instructions you have)
Please separate the instructions into these two categories.